lib/modules/datasource/docker/index.ts

```diff
@@ -234,7 +234,7 @@ export async function getAuthHeaders(
 }
 
 async function getECRAuthToken(
-  region: string | undefined,
+  region: string,
```

Comment:
are we sure region is never undefined here?

Reply:
https://github.com/renovatebot/renovate/blob/1f22ed2a0e4272e285358632e43b85d3799efa98/lib/modules/datasource/docker/index.ts#L105-L115

https://github.com/renovatebot/renovate/blob/1f22ed2a0e4272e285358632e43b85d3799efa98/lib/modules/datasource/docker/index.ts#L52

I think the regex test is sufficient for that.